sdk/agrifood/azure-agrifood-farming/samples/sample_attachments.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+FILE: sample_attachments.py
+
+DESCRIPTION:
+    This sample demonstrates FarmBeats' capabaility of storing arbitrary files 
+    in context to the various farm hierarchy objects. 
+    We first attach some files onto a farmer and a farm, and then download all 
+    existing attachments for the farmer onto a local directory.
+
+USAGE:
+    ```python sample_attachments.py```
+
+    Set the environment variables with your own values before running the sample:
+    - `AZURE_TENANT_ID`
+    - `AZURE_CLIENT_ID`
+    - `AZURE_CLIENT_SECRET`
+    - `FARMBEATS_ENDPOINT`
+"""
+
+from typing import Dict
+from azure.core.exceptions import ResourceNotFoundError
+from azure.identity import DefaultAzureCredential
+from azure.agrifood.farming import FarmBeatsClient
+from azure.agrifood.farming.models import Farmer
+from pathlib import Path
+import os
+
+
+class AttachmentsSample:
+
+    @staticmethod
```

Comment:
sorry i'm going to backpedal and get rid of the sample classes, don't love that you have to set them as `@staticmethod`. V v sorry for telling you the wrong thing to start of with

Reply:
Also, I'd just create the client in the main sample code. My thinking is in the main sample code, i.e. [this](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_alternative_document_input_async.py#L28) function, you should have all of the interesting farmbeasts stuff, including how to set up a client, and how to do everything farmbeats related.

Then, you can have a less intersting `async def main()` and `if __name__ == '__main__'`, where you do the asyncio looping stuff to get it to actually run